### DISCUSSION TITLE
What is the method applied in this package?

### DISCUSSION THREAD

**Discussion by Amuri44** [NREL-SIIP/PowerSimulationsDynamics.jl#223] [Q&A]
What is the applied method in this package to simulate the power network? Is it phasor-domain-type or EMT-type?

**Comment by jd-lara** [comment]
The method depends on the model choice of the user. If you model all lines dynamically you can get an EMT model in the dq0 frame. The model type is determined by the input data. We don't model Waveforms here. 

In terms of methods, this package solves using differential equations solvers from DifferentialEquations.jl

**Reply by Amuri44** [reply]
> The method depends on the model choice of the user. If you model all lines dynamically you can get an EMT model in the dq0 frame. The model type is determined by the input data. We don't model Waveforms here.

I am sorry, I didnt get it well. Can you give me an example for choosing the model based on the input data?

> In terms of methods, this package solves using differential equations solvers from DifferentialEquations.jl

Can you guide me how to run IEEE 39-bus?

**Reply by jd-lara** [reply]
All the models IEEE something are by default quasi-steady-state models. To my knowledge, there are no data sets from standard systems that are set up to do EMT since most machines are set as 2-state or GENROU machines. Moreover, I don't have at hand any IEEE 39 bus data set. I suggest reading the examples in [SIIPExamples](https://github.com/nrel-siip/siipexamples.jl) on how to load and run a PSSe dataset.

**Reply by rodrigomha** [reply]
I'll add that the Network aspect in our documentation how some additional details on which models are used:

https://nrel-siip.github.io/PowerSimulationsDynamics.jl/stable/component_models/network/

**Reply by Amuri44** [reply]
Thanks for your reply and I will refer to your link. For a brief look, I think the method in this package is Phasor Domain.
Actually, I am interested about the results and performance as I worked in EMT-type programing. Please if you were able to guide me running any of standard IEEE systems (such as 39-bus) I will appreciate your assistance.
